examples/common/pigweed/protos/fabric_admin_service.proto

```diff
@@ -6,7 +6,7 @@ package chip.rpc;
 
 // Define the message for a synchronized end device with necessary fields
 message DeviceCommissioningWindowInfo {
-  uint64 node_id = 1;
+  uint64 handle = 1;
```

Comment:
Why not just use ScopedNodeId instead of node_id?

Reply:
This was discussed in the PR's description

![image](https://github.com/user-attachments/assets/ba9f3b3a-dc63-4bdc-a432-1b64eed853da)